tests/foreman/api/test_organization.py

```diff
@@ -266,3 +266,37 @@ def test_negative_update(self, attrs):
         )
         with self.assertRaises(HTTPError):
             response.raise_for_status()
+
+    def test_bugzilla_1103157(self):
```

Comment:
This bug is listed as NEW in bugzilla. Use the `skip_if_bug_open` decorator to conditionally skip the test:

```python
from robottelo.common.decorators import skip_if_bug_open

class SomeTestCase(…):
    @skip_if_bug_open('bugzilla', 1103157)
    def test_bugzilla_1103157(self):
        pass
```